src/App.tsx

```diff
@@ -1,57 +1,56 @@
-import React from 'react';
+import React, { useState, useMemo, useCallback } from 'react';
+import { debounce } from 'lodash';
+
 import './App.scss';
+import Information from './components/Information/Information';
+import DropDownList from './components/DropDownList/DropDownList';
+import { Person } from './types/Person';
+
 import { peopleFromServer } from './data/people';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [currentPerson, setCurrentPerson] = useState<Person | null>(null);
+  const [hasNoResultError, setHasNoResultError] = useState(false);
+  const [appliedQuery, setAppliedQuery] = useState('');
```

Comment:
App component doesn't need appliedQuery state. You can move this logic how to handle this query (and filteredPeople as well) to Dropdown component